### ISSUE TITLE
Menu "☰" button not working for Desktop view

### ISSUE THREAD

**Issue by rfoo1250** [shiven01/asucbc#84]
When viewing from a laptop/desktop (3:2 ratio), the "☰" (`aria-label: Toggle Mobile Menu`) button does not work. On a mobile view or with Developer console open, which I believe was the original intent was for mobile (9:16), it works perfectly fine. Might be an improvement to work for the desktop ratio as well.
## Expected Behavior

On Desktop, clicking the menu button opens the menu where you can see "About", "Team", etc.
## Current Behavior

On Desktop, clicking the menu button does not open the menu.

## Context (Environment)

- Windows 10 Home
- Chrome browser
- 3:2 aspect ratio, full screen
